membership-attribute-service/app/models/DynamoSupporterRatePlanItem.scala

```diff
@@ -15,6 +9,4 @@ case class DynamoSupporterRatePlanItem(
     termEndDate: LocalDate, // Date that this subscription term ends
     contractEffectiveDate: LocalDate, // Date that this subscription started
     cancellationDate: Option[LocalDate], // If this subscription has been cancelled this will be set
-    contributionAmount: Option[BigDecimal],
```

Comment:
I wonder if we should leave this in? The values exist in the data store so it would make sense to make them available here even if we are no longer using them for the current purpose. Or do you think it would be better to remove them from the data store as well?

Reply:
good point, I don't think it hurts to leave the fields in even if they're unused. Maybe we'll use them one day...